kornia/losses/focal.py

```diff
@@ -209,12 +209,12 @@ def binary_focal_loss_with_logits(
     KORNIA_CHECK_IS_TENSOR(pos_weight)
     KORNIA_CHECK(input.shape[-1] == pos_weight.shape[0], "Expected pos_weight equals number of classes.")
 
-    probs_pos = input.sigmoid()
+    probs_pos = torch.sigmoid(input)
```

Comment:
This one was made in purpose towards to "normalise" the library to not only pytorch

Reply:
Thank you for reviewing the changes. Ah, you're saying that `x.sigmoid()` "looks" more "library-independent" and was preferred? I could change the second line to `(-input).sigmoid()` instead and roll back this one if you want...

Reply:
Yeah, that’s right. All rendir libs are converging apis :)

Reply:
Thanks. I switched it over to use the method now :)